Dockerfile.develop

```diff
@@ -15,7 +15,7 @@
 ###############################################################################
 # Create the develop, test, and build environment
 ###############################################################################
-FROM registry.access.redhat.com/ubi8/go-toolset:1.18
+FROM registry.access.redhat.com/ubi8/go-toolset:1.19
```

Comment:
stick with `go-toolset:1.18` for now to avoid having to do a lot more source file updates